src/modules/app.ts

```diff
@@ -1,9 +1,10 @@
 import { Action } from 'redux';
 import { push, CallHistoryMethodAction } from 'connected-react-router';
 import { ThunkDispatch } from 'redux-thunk';
-import { SingleCogniteCapability } from '@cognite/sdk';
+import { SingleCogniteCapability, CogniteClient } from '@cognite/sdk';
 import { RootState } from '../reducers/index';
-import { sdk } from '../index';
+
+export const sdk = new CogniteClient({ appId: 'Discovery' });
```

Comment:
Exporting/importing the SDK from here will couple all the modules and the store implementation making it impossible to test them independently or to provide mock store in a test.

Reply:
moving this out into its own file